uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-fs/src/main/java/org/uberfire/java/nio/fs/file/SimpleFileSystemProvider.java

```diff
@@ -231,16 +218,13 @@ public SeekableByteChannel newByteChannel(final Path path,
         }
     }
 
+    @SuppressWarnings("resource")
     private SeekableByteChannelFileBasedImpl createANewByteChannel(final File file) throws FileNotFoundException {
         return new SeekableByteChannelFileBasedImpl(new RandomAccessFile(file,
-                                                                         "rw").getChannel()) {
-            @Override
-            public void close() throws java.io.IOException {
-                super.close();
```

Comment:
why this is removed?

Reply:
SonarCloud rule violation again.

Overriding methods should do more than simply call the same method in the super class (squid:S1185)

CODE_SMELL Code smell MINOR Minor
Overriding a method just to call the same method from the super class without performing any other actions is useless and misleading. The only time this is justified is in final overriding methods, where the effect is to lock in the parent class behavior. This rule ignores such overrides of equals, hashCode and toString.

Reply:
Reverted back to original style, but SonarCloud might not be happy :(